src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -297,89 +267,164 @@ def build_output_dir(output_dir: Path, subdir: Optional[Union[str, Path]] = None
     return output_dir
 
 
+def get_state_puma_map(df: pd.DataFrame) -> Dict[int, int]:
+    return df.groupby("state")["puma"].apply(lambda x: list(x.unique())).to_dict()
```

Comment:
```suggestion
    return df.groupby("state")["puma"].unique()
```

Do you really need it to be a dict?

Reply:
I can work with something else. What's the issue with dict?

It looks like {state: [pumas]}, eg `{6: [10702, 6514, 3759, ...], 12: [9507, 5301, 5704, ...]}`

Reply:
No issue with dict, but I don't see a reason to prefer it over a pandas Series, which is a datatype more commonly used in Vivarium. The suggestion above makes a pandas Series.

Reply:
And if you do prefer dict, I believe you could still use my suggestion with a `.to_dict()` at the end.

Reply:
Good enough for me, will work on it